contrib/Cargo.toml

```diff
@@ -15,6 +15,12 @@ default = ["json"]
 json = ["serde", "serde_json"]
 tera_templates = ["tera", "templates"]
 handlebars_templates = ["handlebars", "templates"]
+database = []
+diesel_pg = ["diesel/postgres", "r2d2", "r2d2-diesel"]
```

Comment:
I think it would be nice to have each of the implementation features automatically use the `database` feature. The less rocket's users have to add to their Cargo.toml, the better.